platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -161,14 +173,29 @@ BundleContext getBundleContext() {
         return warnings;
     }
 
+    private Collection<Describable> listMigratableDescribables() {
```

Comment:
Excellent question and good observation :-)

We thought we would need the service name in the export and that's something you can't get when you inject a service reference list using blueprint. So we went the complicated way and got the services manually. Well, turns out we didn't need the names so that code could be ripped out from the export method as well and replaced with the injection.